geomesa-core/src/main/scala/org/locationtech/geomesa/core/process/unique/UniqueProcess.scala

```diff
@@ -233,13 +233,11 @@ object AttributeVisitor {
 
   /**
    * Returns a filter that is equivalent to Filter.INCLUDE, but against the attribute index.
-   * Excludes null values, since we don't care about those anyway.
    *
    * @param attribute
    * @return
    */
-  def getIncludeAttributeFilter(attribute: String) =
-    ff.greater(ff.property(attribute), ff.literal(AttributeTable.nullString))
+  def getIncludeAttributeFilter(attribute: String) = ff.greater(ff.property(attribute), ff.literal(""))
```

Comment:
yeah, it ends up acting as a 'prefix' function, e.g. if the rows are:
<sft><attr-name><encoded value>
we create
<sft><attr-name>
which will sort before any encoded value (empty strings are handled as prefixes on AttributeTable#202 and then AttributeIdxStrategy#379)